funnel/utils/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Utility functions."""
+# flake8: noqa
+
+from .markdown import markdown
```

Comment:
Always do `import *` in `__init__.py`, and declare an `__all__` list in the respective modules if you don't want to export everything.